lib/house.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/room'
+
+class House
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price.gsub(/\D/,'').to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room_parameter)
+    @rooms << room_parameter
+  end
+
+  def above_market_average?
+    @market_average = 500000
```

Comment:
You don't _need_ to create a variable for the market average (instance variable, or local), you could write line 18 like this: `if @price > 500000`